apps/go-mod-validator/src/parser.ts

```diff
@@ -0,0 +1,76 @@
+const smartContractKitPrefix = "github.com/smartcontractkit/";
+
+function parseGoModLine(line: string) {
+  // Trim any extra whitespace
+  line = line.trim();
+
+  const moduleRegex = new RegExp(
+    /^([\w.-]+(?:\/[\w.-]+)*) (v\d+\.\d+\.\d+(?:-[\w\d.-]+)?(?:\+[\w\d.-]+)?)$/,
+  );
```

Comment:
Can you include a small comment so we can more easily infer what the regex is trying to accomplish.

Reply:
Done, PTAL